contrib/buildsystems/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ Note: Visual Studio also has the option of opening `CMakeLists.txt`
 directly; Using this option, Visual Studio will not find the source code,
 though, therefore the `File>Open>Folder...` option is preferred.
 
+Visual Studio does not produce a .sln solution file nor the .vcproj files
+that may be required by VS extension tools.
+
+To generate the .sln/.vcproj files run Cmake manually, as below.
```

Comment:
```suggestion
To generate the .sln/.vcproj files run CMake manually, as described below.
```

Reply:
> To generate the .sln/.vcproj files run CMake manually, as described below.

OK.

It's not clear to me if this will be necessary as VS Ninja will run CMakeLists.txt anyway (lots of round incircles interactions, still need to setup the doubly clean testing)